src/main/clojure/portkey/core.clj

```diff
@@ -494,7 +494,7 @@
 (defn deploy! [f lambda-function-name {:keys [keeps environment-variables vpc-config]
                                        :or {environment-variables {}}}]
   (let [bb (-> (java.io.ByteArrayOutputStream.)
-               (doto (package! f keeps))
+               (doto (package! f (seq keeps)))
```

Comment:
Didn't yet figure out why creating a `seq` is necessary here.

Reply:
indeed it's not